README.md

```diff
@@ -47,17 +47,17 @@ After you have installed the CLI, you can run the following from the root of the
 ```bash
 diesel migration run --database-url <DATABASE_URL>
 ```
-Where ```<DATABASE_URL>``` uses the ```postgess://``` URI Example:
-```postgres://username:password@localhost:5432/database_name```
+Where `<DATABASE_URL>` uses the `postgess://` URI Example:
+`postgres://username:password@localhost:5432/database_name`
+
+As this bot deploys on Shuttle, it uses Shuttle secret store and Shuttle async runtime. As such, you need the `cargo-shuttle` crate installed on your local machine to run the bot. You can install this by running: `cargo install cargo-shuttle`
```

Comment:
Whoops, one more thing:

If you reference my comment about this line on the prior review, the command at the end _should_ actually be in triple backticks. A good rule of thumb is that _commands ran by the user_ should be in triple backticks where general package, formatting, or technical information should be in single.

Only thing needed here is to put change `cargo install cargo-shuttle` to:
```bash
cargo install cargo-shuttle
```

Try and reference the format of the steps above and below this to see how they are generally structured as well if possible.

Reply:
This comment rendered as markdown, but to clarify, it should be:

\```bash
cargo install cargo-shuttle
\```

Reply:
fixed it up